integrated_channels/canvas/client.py

```diff
@@ -394,6 +402,7 @@ def _get_oauth_access_token(self, client_id, client_secret):
 
         Returns:
             access_token (str): the OAuth access token to access the Canvas API as the user
+            expires_in (int): the number of seconds after which token will expire
```

Comment:
soooo nitpicky here, but `the token will expire`?

Reply:
:)